charts/prometheus-adapter/values.yaml

```diff
@@ -271,3 +271,6 @@ certManager:
   enabled: false
   caCertDuration: 43800h
   certDuration: 8760h
+
+# Sidecar containers that run alongside prometheus-adapter container.
+sideCarContainers: []
```

Comment:
`extraContainers` is more widely used in this repository. Also, move it above `extraVolumes` so it's more sorted.

```suggestion
# Additional containers to add to the pod
extraContainers: []
```